package.json

```diff
@@ -6,6 +6,9 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
+    "lint": "eslint --max-warnings=0",
+    "test": "jest",
+    "test:watch": "yarn test --watch"
```

Comment:
Here is a typo and also the lint command is duplicated. Also, The yarn install or npm install is broken because this typo

```
error An unexpected error occurred: "/home/andres/Desktop/fcc-group/ecommerce-coffee/package.json: Unexpected string in JSON at position 266".
```

Reply:
The typo, is that there is a missing comma, or remove the following line that is duplicated